requirements_dev.txt

```
@@ -7,7 +7,7 @@ asteval==0.9.25
 click==8.0.3
 netCDF4==1.5.7
 numba==0.54.1
-numpy==1.20.3
+openpyxl==3.0.9
```

```suggestion
numpy==1.20.3
openpyxl==3.0.9
```
I guess removing `numpy` wasn't intended. 😉